src/map.cpp

```diff
@@ -8,23 +8,52 @@
 
 using namespace BitBorn;
 
-Map::Map(std::string mapFilePath, int nMapWidth, int nMapHeight) : nMapWidth(nMapWidth), nMapHeight(nMapHeight) {
+Map::Map(std::string mapFilePath/*, int nMapWidth, int nMapHeight*/) : nMapWidth(nMapWidth), nMapHeight(nMapHeight) {
```

Comment:
This comment should be removed if unused.

Reply:
Hmm, this time there are _two_ lmoa